src/ini_load.cpp

```diff
@@ -226,6 +226,7 @@ void IniLoadFile::LoadFromDisk(const std::string &filename, Subdirectory subdir)
 			s++; // skip [
 			group = &this->CreateGroup(std::string_view(s, e - s));
 			group->comment = std::move(comment);
+			comment.clear(); // std::move leaves comment in a "valid but unspecified state" according to the specification.
```

Comment:
Do we need similar for lines 233 and 248 which do `std::move(comment)` as well?

Reply:
Coverity didn't mention them, but I guess one issue of a kind in a function is enough to bail out. I've added them.